#!/usr/bin/env python
PACKAGE = "franka_experiments"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("admittance", double_t, 0, "Admittance of the controller", 0.0, 0, 0.5)
gen.add("deformed_length", double_t, 0, "N of timestamps for deformation", 0.3, 0, 1)

exit(gen.generate(PACKAGE, "dynamic_compliance", "compliance_param"))
